### PR TITLE
Use primary theming color for country selection

### PR DESCRIPTION
**Pull request by gary-kim**
> Privacy: Country highlight should be primary theming color instead of red

For nextcloud/server#16076

### CODE REVIEW

**Comment by georgehrke** [comment]
@gary-kim Hey, thanks for your contribution! 

When sending pull-requests, that include changes to the user interface, please include before ... after screenshots. :)

File: src/Location.vue [hunk 1]
```diff
@@ -72,7 +72,7 @@ export default {
 				oldElm.style.fill = null
 			}
 			if (newElm) {
-				newElm.style.fill = '#e6605c'
+				newElm.style.fill = 'var(--color-primary)'
```
Comment (georgehrke) [inline src/Location.vue]: @skjnldsv Can we use css vars just like that inside a vue component?

Comment (skjnldsv) [reply src/Location.vue]: Should work yes :)

**Comment by georgehrke** [comment]
@gary-kim Can you please rebase and provide screenshots? Thx!

**Comment by gary-kim** [comment]
> @gary-kim Can you please rebase and provide screenshots? Thx!

Sure! Sorry, I've just been a bit busy recently.
### Before
Screenshot from 2019-07-25 05-30-32
### After
Screenshot from 2019-07-25 05-14-18

**Pull request merged**